repos/emacs/update

```diff
@@ -11,29 +11,27 @@ function update_repo() {
     branch=$3
     echo $repo
 
+    label=${4:-$(echo $branch | sed s/"\/"/"_"/)}
+
     # Get relevant data (commit id and timestamp) for the latest commit
     commit_data=$(curl "https://github.com/$owner/$repo/commits/$branch.atom" | xmlstarlet sel -N atom="http://www.w3.org/2005/Atom" -t -m /atom:feed/atom:entry -v "concat(atom:id,'/',atom:updated)" -n | head -n 1)
-
-    # Extract commit sha and build a version number based on date: YYYYMMDD.0
     commit_sha=$(echo $commit_data | cut -d '/' -f 2)
-    version_number=$(echo $commit_data | cut -d '/' -f 3 | cut -d 'T' -f 1 | sed 's/-//g').0
 
-    output_branch=$(echo $branch | sed s/"\/"/"_"/)
-    digest=$(nix-prefetch-url --unpack "https://github.com/$owner/$repo/archive/${commit_sha}.tar.gz")
-    echo "{\"rev\": \"${commit_sha}\", \"sha256\": \"${digest}\", \"version\": \"${version_number}\"}" > $repo-$output_branch.json
+    { read -r digest; read -r buildPath; } < <(nix-prefetch-url --print-path --unpack "https://github.com/$owner/$repo/archive/${commit_sha}.tar.gz")
+
+    # Grep for Emacs version inside README
+    version=$(cat ${buildPath}/README | grep -Po "This directory tree holds version \K[\d\.]+")
```

Comment:
I'm thinking that something like this would be more solid and less subject to change than a README.

```suggestion
    version=$(grep -Po "GNU Emacs, \K[\d\.]+" ${buildPath}/configure.ac)
```

Reply:
Thanks, couldn't see a solid candidate but configure.ac makes sense :+1: